proposals/0002-explicit-fixity-(~).md

```diff
@@ -0,0 +1,55 @@
+---
+author: Rinat Stryungis
+date-accepted: ""
+ticket-url: "https://gitlab.haskell.org/ghc/ghc/-/issues/18252"
+implemented: "Yes"
+---
+
+This proposal is [discussed at this pull request](https://github.com/haskell-core/core-libraries-proposals/pull/0>).
+**After creating the pull request, edit this file again, update the number in
+the link, and delete this bold sentence.**
+
+# Proposal title
+
+Added explicit fixity for `(~)` and `(~~)`
+
+## Motivation
+
+It solves the issue by adding an explicit fixity for the `(~)` type operator.
+It has been set equal to 4:
+
+    infix 4 ~, ~~
+
+And there are two reasons for that:
+
+1. These fixities are equal to the corresponding fixities of `(:~:)` and `(:~~:)`
```

Comment:
and `(==)`, too.

There's even a type-level `(==)` in `base`:

![image](https://user-images.githubusercontent.com/4061728/92643708-e404a480-f2ea-11ea-8e84-bfa61c00b2a5.png)